### ISSUE TITLE
Ordered dictionaries issues

### ISSUE THREAD

**Issue by MaanooAk** [CSD-FOSS-Team/genume#41]
From the doc:

>Ordered dictionaries are just like regular dictionaries but they **remember the order that items were inserted**.

@Sima214 Was this you original idea, or you wanted something that was keeping the keys sorted?

Also there is some weird problems (i think) from version to version:
```
AttributeError: 'CategoryEntry' object has no attribute '_OrderedDict__map'
```

**Comment by drazioti** [comment]
Python 3.4.2. Καλο είναι να δοκιμασουν όλοι αν τρεχει σωστα για να το δουμε. Στα σχόλια ο  Sima214 γράφει (αρχείο : __main__.py) $python genume/__main__.py (2.6+) και $python -m genume (2.7+).Καλό είναι αυτο να μπει στο README και να διαγραφει η πρωτη εντολή : python genume/__main__.py (2.6+) καθώς και το 2.7+ απο την δευτερη εντολή

**Comment by Sima214** [comment]
The more I look into this bug, the weirder things get...
@drazioti I am uploading a test-fix. Could you check if the **version of python which crashed before** still **crashes** with the new fix?

**Comment by MaanooAk** [comment]
I think we should remove it and replace it a plain Dict.

@Sima214 Can you explain why we need OrderedDict

**Comment by Sima214** [comment]
Seems like 'OrderedDict' is not really meant to be inherited...
Or I just cannot understand how inheritance works on python.

@MaanooAk We need to have the registry entries in an order which makes sense. I think 'OrderedDict' is the most appropriate.

**Comment by MaanooAk** [comment]
Fixed by #46
